src/Utils/api.js

```diff
@@ -1,16 +1,22 @@
 import axios from "axios";
 
-export const getAllArticles = (articleId) => {
-  const url = "https://new-engine.onrender.com/api/articles";
-  return axios.get(url).then(({ data }) => {
+export const getAllArticles = () => {
+  const baseUrl = "https://new-engine.onrender.com/api/articles";
+  return axios.get(baseUrl).then(({ data }) => {
     return { data };
   });
 };
 
 export const getArticleById = (articleId) => {
-  const url = `https://new-engine.onrender.com/api/articles/${articleId}`;
+  const baseUrl = `https://new-engine.onrender.com/api/articles/${articleId}`;
+  return axios.get(baseUrl).then(({ data }) => {
+    return { data };
+  });
+};
 
-  return axios.get(url).then(({ data }) => {
+export const getCommentsByArticleId = (articleId) => {
+  const baseUrl = `https://new-engine.onrender.com/api/articles/${articleId}/comments`;
+  return axios.get(baseUrl).then((data) => {
     return { data };
   });
```

Comment:
Might be worth adding a catch block on all these promise changes, to deal with any errors.